libcxx/include/__functional/function.h

```diff
@@ -74,7 +77,9 @@ class _LIBCPP_EXPORTED_FROM_ABI bad_function_call : public exception {
   const char* what() const _NOEXCEPT override;
```

Comment:
I think it would be good to add why we still consider this an ABI break, since it's not obvious.

Reply:
There was a comment in `__config`, I updated it and corrected it a little.